Updated link to new version of CNTK

bluewatersql opened the pull request:
Tutorial fails because the ResNet model/CNTK version mismatch at Step 2. Updated CNTK install to 2.3

azurecla commented:
Hi __@r-o-b__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Updated link to new version of CNTK

Tutorial fails because the ResNet model/CNTK version mismatch at Step 2. Updated CNTK install to 2.3
